expression/distsql_builtin.go

```diff
@@ -43,11 +43,21 @@ func PbTypeToFieldType(tp *tipb.FieldType) *types.FieldType {
 }
 
 func getSignatureByPB(ctx sessionctx.Context, sigCode tipb.ScalarFuncSig, tp *tipb.FieldType, args []Expression) (f builtinFunc, e error) {
+	if ctx == nil {
+		panic("ctx should not be nil")
+	}
 	fieldTp := PbTypeToFieldType(tp)
-	base, err := newBaseBuiltinFunc(ctx, fmt.Sprintf("PBSig-%v", sigCode), args)
-	if err != nil {
-		return nil, err
+	base := baseBuiltinFunc{
```

Comment:
I think we can use previous code `base, err := newBaseBuiltinFunc(ctx, fmt.Sprintf("PBSig-%v", sigCode), args)`